pkg/disk/disk-unix.go

```diff
@@ -29,3 +29,15 @@ func IsRootDisk(diskPath string) (bool, error) {
 	}
 	return rootDisk, nil
 }
+
+// DiskDeviceID - returns device-id of the mounted disk.
+func DiskDeviceID(diskPath string) (uint64, error) {
+	fi, err := os.Stat(diskPath)
+	if err != nil {
+		return 0, err
+	}
+	if st, ok := fi.Sys().(*syscall.Stat_t); ok {
```

Comment:
is this true on all Unix operating systems? - in-fact isRootDisk might not be same too on all operating systems since .Sys() is platform specific and an interface.. we should test it out properly or say write this only for Linux.

Reply:
just checked in types_unix.go in `os` package, all unixes are supported (except plan9)

Reply:
okay